### ISSUE TITLE
BatchNormalization problem with seperable_con2d

### ISSUE THREAD

**Issue by WassimBouatay** [onnx/tensorflow-onnx#1726]
The exported model gives weird/wrong results for the batchnormalization layer after a seperable_con2d, as you can see in the following image. The mean and variance are zeros and we can see that it somehow uses layer79 instead of layer1. 

![InkedCapture_LI](https://user-images.githubusercontent.com/54542480/134352116-7c86ce82-740f-4fb0-a927-cbbd826a525d.jpg)

Normally the batchnom layer should be fused with the convolution as it was done in this second image: 
![image](https://user-images.githubusercontent.com/54542480/134351542-2d4242ff-b284-425a-baab-fa60c8ce7f08.png)

I am using:
- Tensorflow Version: 1.15
- Python version: 3.6
- Opset 11

Did someone encounter this problem or knows a solution of how to deal with this problem.

Thank you.

**Comment by TomWildenhain-Microsoft** [comment]
Hi @WassimBouatay I'm a bit busy currently but this should be a simple bug to fix hopefully.  If you want to take a shot at it, I'll review your PR.  To find the issue I'd recommend adding some breakpoints and using the python debugger.

**Comment by WassimBouatay** [comment]
@TomWildenhain-Microsoft Thank you for your answer, 

I am actually debugging it but no where near fixing the problem.

**Comment by TomWildenhain-Microsoft** [comment]
@WassimBouatay thanks, really appreciate it. Feel free to post if you have questions.

**Comment by fatcat-z** [comment]
> @TomWildenhain-Microsoft Thank you for your answer,
> 
> I am actually debugging it but no where near fixing the problem.

Could you please share me with your code so I can try to repro it locally for looking for a fix?

**Comment by fatcat-z** [comment]
@WassimBouatay , Could you please share your code so I can repro it locally?
